client/thin-replica-client/include/client/thin-replica-client/grpc_connection.hpp

```diff
@@ -94,7 +94,7 @@ class GrpcConnection {
   // Connect & disconnect from the TRS
   virtual void connect(std::unique_ptr<GrpcConnectionConfig>& config);
   virtual bool isConnected();
-  virtual void reConnect(std::unique_ptr<GrpcConnectionConfig>& config);
+  virtual void checkAndReConnect(std::unique_ptr<GrpcConnectionConfig>& config);
```

Comment:
Nit: In my opinion, `reconnect` is a single word, we can do without the camel-case here.

Reply:
Thanks, @shgandhi , let me change this in next commit.

Reply:
I just realized that after we have the notion of checkAndReconnect, the reconnect is not used and its a private function. So I am removing this function. Thanks for the comment.